### ISSUE TITLE
What's the licensing on the docs?

### ISSUE THREAD

**Issue by trusktr** [meteor/docs#600]
Can I take this and use it as a template for my own docs?

I'm guessing that because there is no license, it automatically means that it is not legal to copy the website.

**Comment by trusktr** [comment]
Suppose I make something like [`Vulcan`](http://docs.vulcanjs.org/). It looks like their docs were copied from an earlier version of Meteor (or at least the visual style is emulated, I'm not sure).

If I made something like that, it could be convenient for users to have all the docs they need in one place.

So if I made something that builds expands Meteor functionality, make a new website for it, could I use Meteor docs in it, to make the whole experience nicer for the user?
